pkg/virtctl/create/vm/vm.go

```diff
@@ -318,7 +306,7 @@ func volumeValidToInferFrom(vm *v1.VirtualMachine, vol *v1.Volume) error {
 		return nil
 	}
 
-	return fmt.Errorf(InvalidInferenceVolumeError)
+	return fmt.Errorf("inference of instancetype or preference works only with datasources, datavolumes or pvcs")
```

Comment:
Why are you keeping some errors as variables and some are returning directly string?

Reply:
In case they are returned only a single time IMO a string is enough. While in case they are returned multiple times I used a const to keep the messages consistent.